### PR TITLE
fix: internal cookie storage isn't reset on test teardown

### PR DESCRIPTION
**Pull request by lesha1201**
@valendres I didn't have time to add tests for that and clean everything up so it's a draft for now. I can add them later. You can push changes directly into PR if you want.

### CODE REVIEW

File: packages/playwright-msw/package.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
     "@playwright/test": ">=1.20.0",
     "msw": ">=0.44.0"
   },
+  "dependencies": {
+    "@mswjs/cookies": "^0.2.2"
+  },
```
Comment (lesha1201) [inline packages/playwright-msw/package.json:32]: Not sure whether it should be a peer dependency or not. I added it as a dependency to not require users to include it in their `package.json`.

Comment (valendres) [reply packages/playwright-msw/package.json:32]: I agree with your reasoning, makes sense for it to be a normal dependency imo.

**Comment by valendres** [comment]
Looks good so far! What kinds of tests are you thinking of writing for this?

If it helps, I'm currently working on getting Jest added so that unit tests can be used in addition to the integration tests.

**Comment by lesha1201** [comment]
@valendres I was planning just to add/change E2E tests in example folder so it covers that case.

**Comment by lesha1201** [comment]
@valendres Sorry, I'm quite busy this week with work so still wasn't able to allocate time for that and I'm not sure I will be able to that on weekend. I would appreciate if you can add the tests. Another option would be to merge it as it is and I will open another PR with tests when I can.

**Comment by valendres** [comment]
@lesha1201 All good mate, I definitely understand being busy! I'm happy to merge this now. I'll spend some time tomorrow morning to add a couple of tests.

**Review by valendres — APPROVED** [review]

**Pull request merged**